cogs/seasonstats.py

```diff
@@ -521,6 +519,129 @@ def __init__(self, bot):
         p = SeasonStatsPaginator(ctx, entries=entries)
         await p.paginate()
 
+    @season_stats.command(name='attacks')
+    async def season_stats_attacks(self, ctx, season: typing.Optional[int] = None):
+        """Get attack wins for all clans.
+
+           By default, you shouldn't need to call these sub-commands as the bot will
+           parse your argument and direct it to the correct sub-command automatically.
+
+           **Example**
+           :white_check_mark: `+season stats attacks`
+        """
+        season = season or await self.bot.seasonconfig.get_season_id() - 1
+        query = """SELECT player_tag, end_attacks - start_attacks as attacks, trophies 
+                        FROM players 
+                        WHERE season_id = $1 AND guild_id = $2
+                        ORDER BY attacks DESC
+                        LIMIT 15
+                    """
+        fetch = await ctx.db.fetch(query, season, ctx.guild.id)
+        table = CLYTable()
+        table.title = f"Attack wins for Season {season}"
+        index = 0
+        for row in fetch:
+            player = await self.bot.coc.get_player(row['player_tag'])
+            table.add_row([index, row['attacks'], player.trophies, player.name])
+        render = table.trophyboard_attacks()
+        fmt = render()
+
+        e = discord.Embed(colour=discord.Colour.gold(), description=fmt)
+        await ctx.send(embed=e)
+
+    @season_stats.command(name='defenses', aliases=['defense', 'defences', 'defence'])
+    async def season_stats_defenses(self, ctx, season: typing.Optional[int] = None):
+        """Get defense wins for all clans.
+
+           By default, you shouldn't need to call these sub-commands as the bot will
+           parse your argument and direct it to the correct sub-command automatically.
+
+           **Example**
+           :white_check_mark: `+season stats defenses`
+        """
+        season = season or await self.bot.seasonconfig.get_season_id() - 1
+        query = """SELECT player_tag, end_defenses - start_defenses as defenses, trophies 
+                        FROM players 
+                        WHERE season_id = $1 AND guild_id = $2
+                        ORDER BY defenses DESC
+                        LIMIT 15
+                    """
+        fetch = await ctx.db.fetch(query, season, ctx.guild.id)
+        defenses = {}
```

Comment:
this could probably rewritten as a dict-comp:
```py
defenses = {n['player_tag']: n['defenses'] for n in fetch}
```

Reply:
done - will push shortly